Formula/zork.rb

```diff
@@ -0,0 +1,30 @@
+class Zork < Formula
+  desc "The Dungeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.0.tar.gz"
+  sha256 "ceedb436cf552bd8ca39abd3e85bae986c3f5df6f31c1bf2806d1b4c8509da67"
+  head "https://github.com/devshane/zork.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "LIBDIR=#{libexec}", "BINDIR=#{bin}"
```

Comment:
`LIBDIR` would usually be the location for library files (`.dylib`, `.a`, `.so` files).

https://www.gnu.org/prep/standards/html_node/Directory-Variables.html

So probably `DATADIR` would be more appropriate, and we'd probably set `DATADIR=#{share}` or `DATADIR=#{pkgshare}` depending on whether collisions can be expected.